contracts/upgradeable_contracts/ERC677BridgeForBurnableMintableToken.sol

```diff
@@ -2,15 +2,22 @@ pragma solidity 0.4.24;
 
 import "./ERC677Bridge.sol";
 import "../interfaces/IBurnableMintableERC677Token.sol";
+import "../libraries/Bytes.sol";
+import "./OtherSideBridgeStorage.sol";
 
-contract ERC677BridgeForBurnableMintableToken is ERC677Bridge {
-    function bridgeSpecificActionsOnTokenTransfer(
-        ERC677 _token,
-        address _from,
-        uint256 _value,
-        bytes /*_data*/
-    ) internal {
+contract ERC677BridgeForBurnableMintableToken is ERC677Bridge, OtherSideBridgeStorage {
+    function chooseReceiver(address _from, bytes _data) internal view returns (address recipient) {
```

Comment:
seems that we have the same method in `amb_erc677_to_erc677/BasicAMBErc677ToErc677.sol`. Does it make sense to have a separate module for it?

Reply:
Moved definition of the method https://github.com/poanetwork/poa-bridge-contracts/pull/302/commits/fd800242b38fcc04946b8b5cceddc3ed4970cf77